mars/api/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+from enum import Enum
```

Comment:
Sorting imports in the right way is a a kinda important thing. The structure:
```python
from __future__ import annotations  # double underscore imports first

from enum import Enum  # then builtins
from os import environ

from django.db import models  # then 3rd party dependencies

from mars.api import serializers  # then local absolute imports

from .views import ExampleView  # and finally local relative ones
```
* 5 groups in a specific order
* every group separated by a blank line
* alphabetic sort within the groups
* `isort` can sort them for you automatically - either install it as a dev dependency in your virtual environment or have it installed globally (since you'll use it *everywhere*). It plays very well with **ALE** in `vim`.
P.s. `views` and `serializers` should both be imported relatively in this case - just separated them to use as examples.